you-dont-need/You-Dont-Need-JavaScript
# Add feature/responsive-product-card

**Description:**
This PR introduces a responsive product card component that showcases product details. Users can hover over the card to reveal additional information about the product, enhancing the user experience and interaction.

## Product Image
Photo by [RAFAEL QUATY](https://www.pexels.com/photo/close-up-photo-of-a-blue-shoe-13580544/) from Pexels.

**Features:**
- Responsive design that adapts to different screen sizes.
- Hover effect to display product details such as description and pricing.
- Clean and modern aesthetic, suitable for any e-commerce application.

**How to Test:**
1. Check out the `feature/responsive-card` branch.
2. Run the application and navigate to the product card section.
3. Hover over the product cards to see the details appear.

**Screenshots:**
###SS1
![Screenshot 2024-10-18 at 4 08 30 PM](https://github.com/user-attachments/assets/d9cbed00-650e-4f1d-a75f-dcba949795f7)
###SS2
![Screenshot 2024-10-18 at 4 08 53 PM](https://github.com/user-attachments/assets/a903d152-85f7-4810-84f5-984549fe3d4d)
